tasks/kmt.py

```diff
@@ -520,7 +612,8 @@ def test(
     if not quick:
         prepare(ctx, stack=stack, vms=vms, ssh_key=ssh_key, full_rebuild=full_rebuild, packages=packages)
 
-    infra = build_infrastructure(stack, ssh_key)
+    ssh_key_obj = try_get_ssh_key(ctx, ssh_key)
+    infra = build_infrastructure(stack, ssh_key_obj)
```

Comment:
Another fix for this task, the TMP path is assumed to match the local host, which is not correct on macOS. Fix:
```
remote_tmp = "/tmp"
remote_run_config = os.path.join(remote_tmp, os.path.basename(tmp.name))
```
```
f"-packages-run-config {remote_run_config}",
```
```
d.copy(ctx, f"{tmp.name}", remote_tmp)
```